Library/Formula/glyr.rb

```diff
@@ -0,0 +1,20 @@
+class Glyr < Formula
+  desc "Music related metadata searchengine with command-line interface and C API"
+  homepage "https://github.com/sahib/glyr"
+
+  url "https://github.com/sahib/glyr/archive/1.0.8.tar.gz"
+  sha256 "0f25f291c7d956bc76d097a4a28595b6607ae8f599988018b36769ef3284b29a"
+
+  depends_on "cmake" => :build
+  depends_on "glib"
+  depends_on "gettext"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/glyrc", "--version"
```

Comment:
To be honest I don't know.  I'm not very familiar with the tool, but it looks as though every function downloads something from the internet and I wasn't sure if that was desired.
I'd be happy to try something.